CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [Unreleased](https://github.com/openfga/go-sdk/compare/v0.6.3...HEAD)
+
+- feat: add support for `start_time` parameter in `ReadChanges` endpoint
+
```

Comment:
@ryanpq can you add the note about the default retry changing to the changelog?

Reply:
changelog updated - Thanks!